clang/lib/Driver/ToolChains/FreeBSD.cpp

```diff
@@ -223,6 +231,12 @@ void freebsd::Linker::ConstructJob(Compilation &C, const JobAction &JA,
       CmdArgs.push_back("--no-relax");
   }
 
+  if (Triple.isLoongArch64()) {
+    CmdArgs.push_back("-X");
+    if (Args.hasArg(options::OPT_mno_relax))
+      CmdArgs.push_back("--no-relax");
```

Comment:
A sentence or two describing the justification for the (hopefully temporary) disabling could be better.

Reply:
> A sentence or two describing the justification for the (hopefully temporary) disabling could be better.
here, I just construct the clang linker options from cmdline args, it's up to the user to decide whether to turn it on or not

Reply:
Ah I missed the `hasArg` part. I have no problem with it then...

Reply:
> Ah I missed the `hasArg` part. I have no problem with it then...

Thanks !